kolibri/plugins/learn/assets/src/vue/content-page/index.vue

```diff
@@ -16,9 +16,10 @@
       :extra-fields="content.extra_fields">
     </content-renderer>
 
-    <icon-button v-link="nextContentLink" v-if="progress >= 1 && showNextBtn" class="next-btn">
-    {{ $tr("nextContent") }}
-    <svg class="right-arrow" src="../icons/arrow_right.svg"></svg></icon-button>
+    <icon-button @click="nextContentClicked" v-if="progress >= 1 && showNextBtn" class="next-btn">
```

Comment:
The `icon-button` is implemented as a `<button>`, not an `<a>`. I actually don't know how this worked before when the `v-link` was attached to it.

----

note - I still feel that perhaps 'next content' would be better implemented as a link because it navigates to a new context, rather than changing something in the current context. cc @66eli77 @rayykim